packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -316,6 +328,9 @@ class MessageList extends Widget<Properties> {
     this._createMessageGroupComponent([message], author?.id);
 
     this._scrollContentToLastMessage();
+
+    lastMessageGroup = this._messageGroups?.at(-1);
+    lastMessageGroup?._updateIsLastOptionOfLastMessage(true);
```

Comment:
[excess template call]
That's not good that we we render last bubble twice: with `isLast=false` and then with `isLast=true`

Reply:
0e11dd012baad48fce62eeab905c60abc39aa9d5